clients/src/main/java/org/apache/kafka/common/network/Selector.java

```diff
@@ -466,8 +466,10 @@ public void poll(long timeout) throws IOException {
         long startSelect = time.nanoseconds();
         int numReadyKeys = select(timeout);
         long endSelect = time.nanoseconds();
+        log.trace("Completed select in " + (endSelect - startSelect) + "ms");
```

Comment:
general concerns about such logs: since the client will be calling this poll in a tight loop (producer in a while true loop and consumers fetch offset same), and there're many producers/consumers in venice server, not to mention eventBus, these logs in selector might flood the log file in seconds, not sure if it can capture a timeout case before anything crashes.

Reply:
Yes, that's a valid concern. I hope it's still manageable.